experimental/segmenter/Cargo.toml

```diff
@@ -34,7 +34,7 @@ serde = { version = "1.0", default-features = false, features = ["derive", "allo
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
```

Comment:
`serde_json` is hopefully test-only as well?

Reply:
Ah, after moving data to Data Provier, it is unnecessary now. I should move it to `dev-dependencies`.